Automattic/wp-calypso
# Jetpack Licensing Portal - Exclude certain products from the license issue screen

We will introduce new products (and a few superseding products) to the Start API in D82300-code, and this patch will need to ship in parallel in order to keep our licensing portal offering accurate.

#### Context
* PT: pdpAdu-kD-p2
* 1201801459945917-as-1202490864329209 (this)
* 1201801459945917-as-1202490864329203 (parallel patch)

#### Proposed Changes

* Exclude backup-daily, it will be superseded by backup-tier-1
* Exclude backup-realtime, it will be superseded by backup-tier-2
* Exclude backup-t0, it will not be offered via the Licensing Portal
* Exclude security-daily, it will be superseded by security-tier-1
* Exclude security-realtime, it will be superseded by security-tier-2
* Exclude jetpack-boost, it will be offered soon but is not ready quite yet

#### Testing Instructions

* Spin up this branch
* Apply D82300-code to your sandbox
* Point the API to your sandbox
* Go to `http://jetpack.cloud.localhost:3000/partner-portal/issue-license`
* Verify that you see the following products/plans:
  * `Anti-Spam`
  * `Backup 10GB`
  * `Backup 1TB`
  * `Complete`
  * `Scan Daily`
  * `Security 10GB`
  * `Security 1TB`
  * `VideoPress`
